src/diff.js

```diff
@@ -29,6 +29,10 @@
  *
  * QUnit.diff( "the quick brown fox jumped over", "the quick fox jumps over" ) === "the  quick <del>brown </del> fox jump<ins>s</ins><del>ed</del over"
  */
+
+// Don't load QUnit.diff on non-Browser environments
+if ( typeof window !== "undefined" && window.document ) {
```

Comment:
> Can't we just move the diff module inside the intro/outro closure?

yes, sure.

I would rather avoid this and apply something solid to QUnit.diff for node environments.